roles/smallfile-bench/templates/workload_job.yml.j2

```diff
@@ -40,24 +40,23 @@ spec:
           - name: uuid
             value: "{{ uuid }}"
           - name: test_user
-            value: "{{ test_user }}"
+            value: "{{ test_user | default("ripsaw") }}"
           - name: clustername
             value: "{{ clustername }}"
-{% if elasticsearch.server is defined %}
           - name: es
-            value: "{{ elasticsearch.server }}"
+            value: "{{ elasticsearch.server | default() }}"
           - name: es_port
-            value: "{{ elasticsearch.port }}"
+            value: "{{ elasticsearch.port | default() }}"
           - name: es_index
```

Comment:
why do you need to have es_index if value is removed?   Why not just let es_index default to ripsaw-smallfile?  We know we are in ripsaw at this point.   This is one of most annoying parts of smallfile CR at this point, would like to get rid of it.

Reply:
I don't understand. It's already set to "ripsaw-smallfile"

Reply:
withdrawing this comment, it is really another case of the same comment up above that you answered.